tests/e2e/task2.test.js

```diff
@@ -0,0 +1,19 @@
+import {test, expect} from '@playwright/test'
+
+test.describe("TodoList homepage verifications", () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto("http://localhost:3030/")
+  })
+
+  test("A List can be added by using Add button in the List header", async ({page}) => {
```

Comment:
This test misses final expectations. For example, if Add button is clicked but it does not work properly and the list is not added, we won't know about it.